src/components/HistoryRecord/HistoryRecord.tsx

```diff
@@ -153,10 +153,14 @@ const componentMap: {
                         <ChangeListItem title={tr('Login')} after={event.after.login} />
                         <ChangeListItem title={tr('Team')} after={event.after.groupId} />
                         <ChangeListItem title={tr('Organization id')} after={event.after.organizationUnitId} />
+                        <ChangeListItem title={tr('Unit id')} after={event.after.unitId} />
```

Comment:
add coordinators

Reply:
I put coordinator ids and logins to coordinator id and login, planning to migrate to multiple coordinators completely